src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
+
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = !this.hidden;
```

Comment:
Do you need to unhide in second call?

Reply:
"Create a Herbivore class. Herbivore has a method of hide, which changes the hidden property of the beast to the opposite value and helps to hide from carnivores"
Is it not right? I thought we need just the opposite value?

Reply:
Good point, but we need just hide herbivores))

Reply:
Ok, I'll fix it)